lib/authentic/key_manager.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require 'json/jwt'
+require 'unirest'
+require 'uri'
+require File.expand_path('key_store', __dir__)
```

Comment:
As mentioned in DM, for all these requires, `require 'authentic/key_store'`. Using the file module here may work for development, but it could blow up terribly in certain circumstances. The correct way to make this always work is to ensure that the `lib` directory is in the `LOAD_PATH`, which you do in your gemspec already. `irb` won't automatically set up the `LOAD_PATH` for you when doing development in the directory, but consumers will have the path set up correctly.

Reply:
Not sure what I was thinking there. Yea this doesn't make sense. Thanks for the `bundle console` tip 👍 65f9fc697545af4eb109f90fdb36d27a94d9f1c2